modules/player/build.gradle.kts

```diff
@@ -1,3 +1,6 @@
-plugins {
-    `java-library`
-}
+apply(plugin = "java-library")
```

Comment:
plugin blocks should be favored over `apply(plugin = "...")` for builtin gradle plugins (`java`, `java-library`, `application`, etc) because IntelliJ recognizes some of them to add their associated config blocks. Using the apply syntax, that will not be recognized and you have to figure out the import for them